tdm-reader/nitdmreader.yml

```diff
@@ -0,0 +1,508 @@
+basePath: /nitdmreader
+consumes:
+  - application/json
+produces:
+  - application/json
+securityDefinitions:
+  basicAuth:
+    type: basic
+  cookieAuth:
+    type: apiKey
+    in: header
+    name: Cookie
+security:
+  - basicAuth: []
+  - cookieAuth: []
+x-ni-routing-key: Skyline.TDMReader
+definitions:
+  Error:
+    description: Contains error information
+    type: object
+    properties:
+      name:
+        description: String error code
+        type: string
+      code:
+        description: Numeric error code
+        type: integer
+      resourceType:
+        description: Type of resource associated with the error
+        type: string
+      resourceId:
+        description: Identifier of the resource associated with the error
+        type: string
+      message:
+        description: Complete error message
+        type: string
+      args:
+        description: Positional argument values for the error code
+        type: array
+        items:
+          type: string
+      innerErrors:
+        type: array
+        items:
+          $ref: '#/definitions/Error'
+    example:
+      name: Skyline.OneOrMoreErrorsOccurred
+      code: -251040
+      message: >-
+        One or more errors occurred. See the contained list for details of each
+        error.
+      args: []
+      innerErrors:
+        - name: TDMReader.FileServiceFailed
+          code: -253500
+          resourceType: File
+          resourceId: 4afb2ce3741fe11d88838cc9
+          message: File service failed.
+          args: []
+  Operation:
+    description: An operation provided by the API
+    type: object
+    properties:
+      available:
+        type: boolean
+        description: Whether the operation is available to the caller
+      version:
+        type: integer
+        description: The version of the available operation
+    required: [available]
+    example:
+      available: true
+      version: 1
+  V1Operations:
+    description: V1 operations
+    type: object
+    properties:
+      operations:
+        description: >-
+          Available operations in the v1 version of the API:
+
+          - readData: The ability to read data
+        type: object
+        properties:
+          readData:
+            $ref: '#/definitions/Operation'
+  Property:
+    description: A property and its value.
+    type: object
+    properties:
+      name:
+        type: string
+        description: name of the property
+        example: Name
+      type:
+        type: string
+        description: data type of 'value'
+        example: String
+      value:
+        type: string
+        example: Channel name
+  Properties:
+    properties:
+      properties:
+        description: Property values.
+        type: array
+        items:
+          $ref: '#/definitions/Property'
+      totalCount:
+        type: integer
+        description: Count of all properties. This could be more than the items returned in Properties.
+        example: 1
+  FileList:
```

Comment:
This doesn't have the latest change made last Thursday that wraps this in an object.